src/nvapi/nvapi_adapter.cpp

```diff
@@ -268,6 +268,10 @@ namespace dxvk {
             || (HasNvkDriver() && IsVkDeviceExtensionSupported(VK_EXT_SHADER_VIEWPORT_INDEX_LAYER_EXTENSION_NAME)))
             return NV_GPU_ARCHITECTURE_GM200;
 
+        // VK_EXT_shader_image_atomic_int64 is supported on Maxwell 1 (GM10x) and newer
```

Comment:
This is also the case for NVK (https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/src/nouveau/vulkan/nvk_physical_device.c?ref_type=heads#L244) which is nice!

Reply:
This is the soure ;) so no double check.
As double check I filtered all devices by this thing on [vulkan.gpuinfo.org](https://vulkan.gpuinfo.org/listdevicescoverage.php?extension=VK_EXT_shader_image_atomic_int64&platform=linux) with additional finter for NV -- only gen1 maxwells and some weird setups was lowest.